pysrc/SBProfile.cpp

```diff
@@ -79,6 +79,9 @@ namespace galsim {
                 .def(bp::self == bp::other<GSParams>())
                 .enable_pickling()
                 ;
+#if BOOST_VERSION >= 106000 && BOOST_VERSION < 106100
```

Comment:
Do you really think we want the `< 106100` part of this?  Is this something they've already changed back in version 1.61?  I would have assumed that this line will be required for all versions >= 1.60.